### PR TITLE
Use cmdline option --print_events

### PR DESCRIPTION
**Pull request by mkanoor**
--print-events allows for the printing of events as they are being read from the source queue.
If --debug is used --print-events is disabled, since --debug logs the event already.

```
ansible-rulebook --inventory ./tests/playbooks/inventory.yml -S ./tests/sources --rulebook rule1.yml --print-events
{   'address': {'city': 'Bedrock', 'state': 'NJ', 'street': '123 Any Street'},
    'age': 10,
    'name': 'Fred Flintstone'}
Shutdown()
```

### CODE REVIEW

File: ansible_rulebook/cli.py [hunk 1]
```diff
@@ -134,6 +135,11 @@ def get_parser() -> argparse.ArgumentParser:
         "--controller-token",
         help="Controller API authentication token",
     )
+    parser.add_argument(
+        "--print_events",
```
Comment (ttuffin) [inline ansible_rulebook/cli.py:139]: To remain consistent with the other args I think this should be `--print-events`.

**Review by ttuffin — CHANGES_REQUESTED** [review]
If both `--print_events` `--debug` are used together the output is printed twice:
```
2023-01-23 08:36:13,907 - ansible_rulebook.engine - DEBUG - None
{'i': 4}
2023-01-23 08:36:13,907 - ansible_rulebook.engine - DEBUG - Received event : {'i': 4}
```

Is this the intended behavior? For brevity, it might be better to suppress print-events when --debug is used?

**Comment by mkanoor** [comment]
@Alex-Izquierdo @ttuffin Changes made please review

**Review by ttuffin — APPROVED** [review]

**Review by Alex-Izquierdo — APPROVED** [review]
lgtm

**Pull request merged**